Main/ClientApp/src/Stores/CollectionsStore.tsx

```diff
@@ -12,6 +12,10 @@ class CollectionStore extends StoreBase {
         this.emitChange();
         break;
       }
+      case ActionTypes.ADD_COLLECTION: {
+        loadCollections();
```

Comment:
Catching a ADD_COLLECTION action isn't the store's responsibility. Comment re. that in  the API